Library/Formula/boxes.rb

```diff
@@ -17,7 +17,7 @@ def install
 
     # distro uses /usr/share/boxes change to prefix
     system "make",
-      "GLOBALCONF=#{share}/boxes-config",
+      "GLOBALCONF=#{pkgshare}/boxes-config",
```

Comment:
By all means if you want to play with the audit check to iron out these situations be my guest :)